### ISSUE TITLE
"Cannot use 'in' operator to search for 'selectionStart' in null"

### ISSUE THREAD

**Issue by ColemanGariety** [sarajs/Sara#54]
Keydown events on input/textareas throw this error:

```
Cannot use 'in' operator to search for 'selectionStart' in null
```

Been around for a while. It's resolved by setting e.srcElement to the document node, but that causes react to lose its place on the next step. Not sure what's happening here.

**Comment by ColemanGariety** [comment]
I've resolved it on keydown of inputs in 2ecb5bce14 by adding the focus event—but now it happens when you keydown on the document body, but now it happens everywhere except for input elements.

**Comment by ColemanGariety** [comment]
Resolved in 7bd94d1550 of https://github.com/sarajs/jsdom
